android/src/main/java/com/firebasePushNotification/FirebasePushNotification.java

```diff
@@ -68,6 +71,24 @@ public void checkPermissionForPushNotification(Callback callback) {
         callback.invoke(null,NotificationManagerCompat.from(getCurrentActivity()).areNotificationsEnabled());
     }
 
+
+    @ReactMethod
+    public void getToken(final Callback callback) {
+        FirebaseInstanceId.getInstance().getInstanceId()
```

Comment:
FirebaseInstanceId is deprecated, we should use FirebaseMessaging class instead.